crates/testing/src/overall_safety_task.rs

```diff
@@ -253,7 +253,7 @@ pub struct RoundResult<TYPES: NodeType> {
     /// block -> # entries decided on that block
     pub block_map: HashMap<VidCommitment, usize>,
 
-    /// node idx -> number transactions
+    /// number transactions -> node idx
```

Comment:
How come this order changed? Also, nit: "number of transactions"

Reply:
The documentation was just incorrect. That sad, it's still wrong, it's not node index but number of nodes reporting this number of transactions, I'll make amends.